src/pages/list/search/articles/index.tsx

```diff
@@ -0,0 +1,267 @@
+import { Icon as LegacyIcon, Form } from '@ant-design/compatible';
+import { Button, Card, Col, List, Row, Select, Tag } from 'antd';
+import React, { Component } from 'react';
+
+import { Dispatch } from 'redux';
+import { FormComponentProps } from 'antd/es/form';
+import { connect } from 'dva';
+import ArticleListContent from './components/ArticleListContent';
+import { StateType } from './model';
+import { ListItemDataType } from './data.d';
+import StandardFormRow from './components/StandardFormRow';
+import TagSelect from './components/TagSelect';
+import styles from './style.less';
+
+const { Option } = Select;
+const FormItem = Form.Item;
+
+const pageSize = 5;
+
+interface ArticlesProps extends FormComponentProps {
+  dispatch: Dispatch<any>;
+  listAndsearchAndarticles: StateType;
+  loading: boolean;
+}
+
+class Articles extends Component<ArticlesProps> {
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch({
+      type: 'listAndsearchAndarticles/fetch',
+      payload: {
+        count: 5,
+      },
+    });
+  }
+
+  setOwner = () => {
+    const { form } = this.props;
+    form.setFieldsValue({
+      owner: ['wzj'],
+    });
+  };
+
+  fetchMore = () => {
+    const { dispatch } = this.props;
+    dispatch({
+      type: 'listAndsearchAndarticles/appendFetch',
+      payload: {
+        count: pageSize,
+      },
+    });
+  };
+
+  render() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class method 'render' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)